client/src/components/Login.js

```diff
@@ -1,57 +1,79 @@
-            errors: {},
-            formSubmitted: false,
+            email: '',
+            passwort: '',
+
 
         }
+        this.handleEmailChanged = this.handleEmailChanged.bind(this);
+        this.handlePasswortChanged = this.handlePasswortChanged.bind(this);
+        this.verifyUserInput = this.verifyUserInput.bind(this);
+
+
     }
 
-    handleInputChange = (event) => {
-        const target = event.target;
-        const value = target.value;
-        const name = target.name;
+    handleEmailChanged(event) {
+        this.setState({email: event.target.value});
 
-        let { formData } = this.state;
-        formData[name] = value;
 
-        this.setState({
-            formData: formData
-        });
     }
 
+    handlePasswortChanged(event) {
+        this.setState({passwort: event.target.value});
+
+    }
+
+    verifyUserInput = async (event) => {
+        event.preventDefault();
+        const datatest = await userService.submitUser(this.state.email, this.state.passwort);
+        console.log(datatest);
+
+
+        if (datatest.user != null) {
+            localStorage.setItem('email', JSON.stringify(this.state.email));
+            this.props.history.push('/');
+        }
+
+        else {
+            console.log("no matching")
+        }
+    };
+
+
     render() {
 
-        return (
+        return(
             <div className="Login">
-                <Row>
-            <form onSubmit={this.login}>
-    <FormGroup controlId="email">
-            <FormLabel>E-mail</FormLabel>
-            <FormControl type="email" name="E-mail" placeholder="email" onChange={this.handleInputChange} />
-    </FormGroup>
-        <FormGroup controlId="passwort" >
-            <FormLabel>Passwort</FormLabel>
-            <FormControl type="password" name="passwort" placeholder="passwort" onChange={this.handleInputChange} />
-    </FormGroup>
-        <Button type="submit" bsStyle="primary">Log In</Button>
-        </form>
-        </Row>
-        </div>
-    )
+                        <form onSubmit={this.verifyUserInput} >
+                            <h3 align="center" >Willkommen beim Adesso TechnologieRadar</h3>
+                            <FormGroup controlId="email">
+                                <input type="email" name="email" placeholder="email" onChange={this.handleEmailChanged}/>
+                            </FormGroup>
+                            <FormGroup controlId="passwort">
+                                <FormControl type="password" name="passwort" placeholder="passwort"
+                                             onChange={this.handlePasswortChanged}/>
+                            </FormGroup>
+                            <FormGroup controlId="submit">
+                                <Button type="submit" onClick={this.verifyUserInput} bsstyle="primary">Einloggen</Button>
+                            </FormGroup>
+                        </form>
+                }
+            </div>
+        );
```

Comment:
nothing important .. just added some methods I e onClick={this.verifyUserInput} and so on